WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/Tap to Pay Education/TapToPayEducationViewModel.swift

```diff
@@ -0,0 +1,202 @@
+import Foundation
+import Yosemite
+
+final class TapToPayEducationViewModel: ObservableObject {
+    struct Action {
+        let title: String
+        let action: () -> Void
+    }
+
+    enum Flow {
+        case onboarding
+        case about
+    }
+
+    @Published var selectedStep = 0
+    @Published var steps: [TapToPayEducationStepViewModel]
+    @Published private(set) var isInteractiveDismissDisabled = false
+
+    @Published var showingSetUpFlow: Bool = false
+    @Published private(set) var hasPreviousTapToPayUsage = false
+
+    private let flow: Flow
+    private let cardReaderSupportDeterminer: CardReaderSupportDetermining
+
+    let configuration: CardPresentPaymentsConfiguration
```

Comment:
'Set up Tap to Pay' logic is taken from `AbouTapToPayView`, since we want to show an entry point to setting up the TTP when education is opened from 'About Tap to Pay' menu option